src/finn/transformation/fpgadataflow/floorplan.py

```diff
@@ -70,7 +70,7 @@ def apply(self, model):
 
         try:
             default_slr = self.user_floorplan["Defaults"]["slr"][0]
-        except:
+        except Exception:
```

Comment:
@maltanar I forgot to mention this change, which was required by the pre-commit hooks. This change will still catch almost all exceptions, excluding: `SystemExit`, `KeyboardInterrupt` and `GeneratorExit`. And I don't think we really want to catch those excluded exceptions here.

Reply:
Good catch (pun intended) -- here we would primarily be interested in the cases where there's no specified floorplan entry. I think what the original author (@quetric ) is trying to capture here is that the exception may be of various types (e.g. `user_floorplan` may be None, or it may not have the defaults set up). Let's go with this solution.